cloud-aws-common/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/common/AwsSubnetIgwExplorer.java

```diff
@@ -65,11 +70,41 @@ private Optional<RouteTable> getImplicitRouteTable(List<RouteTable> routeTables)
     }
 
     private Optional<Route> getRouteWithInternetGateway(Optional<RouteTable> routeTable) {
```

Comment:
please rename the private method since the logic started to support the VPCE besides the InternetGateway